frontend/src/hooks/useDashboardCreateForm/index.tsx

```diff
@@ -40,14 +41,49 @@ const initialRecruitmentInfoState: RecruitmentInfoState = {
 };
 
 export default function useDashboardCreateForm(): UseDashboardCreateFormReturn {
-  const [stepState, setStepState] = useState<StepState>('recruitmentForm');
-  const [recruitmentInfoState, setRecruitmentInfoState] = useState<RecruitmentInfoState>(initialRecruitmentInfoState);
-  const [applyState, setApplyState] = useState<Question[]>(DEFAULT_QUESTIONS);
+  const clubId = useClubId().getClubId() || '';
+  const LOCALSTORAGE_KEYS = {
+    STEP: `${clubId}-step`,
+    INFO: `${clubId}-info`,
+    APPLY: `${clubId}-apply`,
+  } as const;
+
+  const [enableStorage] = useState(() => {
+    const Step = window.localStorage.getItem(LOCALSTORAGE_KEYS.STEP);
+    const Info = window.localStorage.getItem(LOCALSTORAGE_KEYS.INFO);
+    const Apply = window.localStorage.getItem(LOCALSTORAGE_KEYS.APPLY);
+
+    if (Step || Info || Apply) {
+      return window.confirm('이전 작성중인 공고기 있습니다. 이어서 진행하시겠습니까?');
```

Comment:
멋진 로직에 감탄하던 중 발견한 오타가 😅 

```suggestion
      return window.confirm('이전 작성중인 공고가 있습니다. 이어서 진행하시겠습니까?');
```